code/modules/projectiles/guns/energy.dm

```diff
@@ -594,3 +594,393 @@
 	A.update_hud(user)
 
 	return TRUE
+
+// Neat laser gun things
+
+//Ok the rifle
+
+/obj/item/weapon/gun/energy/lasgun/standard_marine
+	name = "\improper TEX-R Lasrifle"
```

Comment:
```suggestion
	name = "\improper TEX-R lasrifle"
```

Is "TEX" the same as "Terragov EXperimental" or is it "**T**erragov **E**nergy e**X**perimental?"